autogpt/ai_functions.py

```diff
@@ -1,8 +1,10 @@
-from typing import List, Optional
 import json
-from config import Config
-from call_ai_function import call_ai_function
-from json_parser import fix_and_parse_json
+from typing import List
```

Comment:
Don't need to import Optional from typing? (legit asking just because I see it was removed from the original)